src/test/kotlin/com/igorwojda/list/formattrainroute/challenge.kt

```diff
@@ -4,10 +4,18 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun formatTrainRoute(stations: List<String>): String {
-    TODO("not implemented")
+    var text = "Train is calling at "
+
+    if (stations.size >= 2) {
```

Comment:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/join-to-string.html is a very powerful tool

Reply:
reimplemented using joinToString